java/compaction/compaction-core/src/test/java/sleeper/compaction/testutils/InMemoryCompactionJobStatusStore.java

```diff
@@ -94,13 +94,9 @@ public void jobFinished(CompactionJobFinishedEvent event) {
 
     @Override
     public void jobCommitted(CompactionJobCommittedEvent event) {
-        jobCommitted(event, getUpdateTimeOrDefault(Instant::now));
-    }
-
-    public void jobCommitted(CompactionJobCommittedEvent event, Instant committedTime) {
         add(event.getTableId(), ProcessStatusUpdateRecord.builder()
                 .jobId(event.getJobId()).taskId(event.getTaskId()).jobRunId(event.getJobRunId())
-                .statusUpdate(CompactionJobCommittedStatus.committedAt(committedTime))
+                .statusUpdate(CompactionJobCommittedStatus.committedAt(event.getCommitTime(), defaultUpdateTime(event.getCommitTime())))
```

Comment:
I think this should use getUpdateTimeOrDefault like the other methods here, otherwise fixUpdateTime won't work for commit events. It's possible we'll remove the update time completely at some point though.

Reply:
Thanks, fixed